routes/api/v1/favorites.js

```diff
@@ -41,4 +41,22 @@ router.post('/', (request, response) => {
       });
 });
 
+router.get('/:id', (request, response) => {
+  let id = request.params.id
+  database('favorites').where('id', id)
+  .select('id', 'title', 'artistName', 'genre', 'rating')
+  .then(favorite => {
+    if (favorite.length) {
+      response.status(200).json(favorite[0]);
```

Comment:
Do you think we should assign this to `favorite[0]` to a variable and then call it here?

Reply:
Ya we could definitely do that!